server/wavelength/modules/economy/index.js

```diff
@@ -0,0 +1,180 @@
+// Used with nef
+
+'use strict';
+
+/** @type {typeof import('../../../../lib/fs').FS} */
+const FS = require('../../../../.lib-dist/fs').FS;
+const DB_PATH = 'config/database'; // path to your database folder
+
+module.exports = class Economy {
+	constructor(name, dbName) {
+		this.name = name;
+		this.dbName = dbName;
+		this.db = Db[dbName];
+
+		this.transactionLogs = FS('logs/transactions.txt');
+		this.tLogs = this.transactionLogs.createAppendStream();
+	}
+
+	/**
+	 * Returns the currency label
+	 * @param {number} value
+	 * @param {boolean} [includeValue=true]
+	 * @return {string}
+	 */
+
+	getCurrencyLabel(value, includeValue = true) {
+		return `${includeValue ? value : ''} ${this.name}${Chat.plural(value)}`;
```

Comment:
The problem with this line is that if `includeValue` is falsy, then it will return the currency name with a space before it. This would need to be corrected to the following to work properly if you absolutely must use a one-liner:
```js
getCurrencyLabel(value, includeValue = true) {
   return `${includeValue ? `${value} ` : ''}${this.name}${Chat.plural(value)}`;
}
```
However, it is not the most readable, which is why I proposed the original function definition. The issue about readability, however, is but a nitpick.

Reply:
I don't think readability would be an issue for this? and mb, i switched it over from what you proposed to that, didn't account for the `space` if it's false